instructor/templates/instructor/assignment_sidebar.html

```diff
@@ -0,0 +1,131 @@
+{% extends "instructor/assignment_edit_base.html" %}
+
+{% block sidebar %}
+    <div style="width:200px; display:block;float:left">
+
+        <div class='scb_s_instructor_sidebar_section_heading' role='heading'>Assignment Setup</div>
+
+        {% if new %}
+            <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+                <a class='scb_s_assignment_sidebar_link' href="{% url "common_assignment_setup" %}">
+                    Assignment setup
+                </a>
+            </div>
+            <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+                <a class='scb_s_assignment_sidebar_link' href="{% url "common_course_setup" %}">
+                    Course setup
+                </a>
+            </div>
+        {% else %}
+            <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+                <a class='scb_s_assignment_sidebar_link' href="{% url "common_assignment_modify" %}">
+                    Assignment setup
+                </a>
+            </div>
+            <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+                <a class='scb_s_assignment_sidebar_link' href="{% url "common_course_modify" %}">
+                    Course setup
+                </a>
+            </div>
+        {% endif %}
+
+        <div class='scb_s_instructor_sidebar_section_heading' role='heading'>Experiment Setup</div>
+        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+            <a class="scb_s_assignment_sidebar_link" href="{% url "common_assignments_edit_strains" %}">
+                Strains
+            </a>
+        </div>
+        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+            <a class='scb_s_assignment_sidebar_link'
+               href="{% if not new %}{% url "common_assignments_edit_meta" %}{% endif %}">
+                Experimental variables
+            </a>
+        </div>
+        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+            <a class="scb_s_assignment_sidebar_link"
+               href="{% if not new %}{% url "common_assignments_edit_text" %}{% endif %}">
+                Text
+            </a>
+        </div>
+
+        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+            <a class="scb_s_assignment_sidebar_link"
+               href="{% if not new %}{% url "common_assignments_edit_protocols" %}{% endif %}">
+                Protocols
+            </a>
+        </div>
+
+        {% for pd in assignment.protocols.all %}
```

Comment:
protocol? I don't know what pd is supposed to mean, but I would think protocol, what is the d?